modules/auxiliary/scanner/rdp/cve_2019_0708_bluekeep.rb

```diff
@@ -31,6 +32,11 @@ def initialize(info = {})
         ],
       'DisclosureDate' => '2019-05-14',
       'License'        => MSF_LICENSE,
+      "Actions" => [
+        ["Scan", "Description" => "Scan for exploitable targets"],
+        ["Crash", "Description" => "Trigger denial of service vulnerability"],
+      ],
+      "DefaultAction" => "Scan",
       'Notes'          =>
         {
           'Stability' => [ CRASH_SAFE ],
```

Comment:
lol

Reply:
I think I know why it doesn't crash sometimes... so when it's fixed it should Safely Crash on command ;)